spec/services/people/importer_spec.rb

```diff
@@ -0,0 +1,111 @@
+RSpec.describe People::Importer do
+  subject(:service) { described_class.new(nomis_attributes) }
+
+  let(:nomis_attributes) do
+    {
+      prison_number: 'G8872UH',
+      latest_booking_id: 1_093_139,
+      last_name: 'JAVAS',
+      first_name: "EF'LIAICO",
+      middle_names: 'MATTHEWS BARNARDOS',
+      date_of_birth: '1989-10-01',
+      aliases: nil,
+      pnc_number: '05/886838E',
+      cro_number: '610592/05U',
+      gender: gender.nomis_code,
+      ethnicity: ethnicity.title,
+      nationalities: 'British',
+    }
+  end
+  let(:gender) { create(:gender) }
+  let(:ethnicity) { create(:ethnicity) }
+
+  describe '#call' do
+    let(:now) { Time.zone.now }
+
+    let(:expected_person_attributes) do
+      {
+        'id' => nil,
+        'created_at' => nil,
+        'updated_at' => nil,
+        'nomis_prison_number' => 'G8872UH',
+        'prison_number' => 'G8872UH',
+        'criminal_records_office' => '610592/05U',
+        'police_national_computer' => '05/886838E',
+        'first_names' => "EF'LIAICO MATTHEWS BARNARDOS",
+        'last_name' => 'JAVAS',
+        'date_of_birth' => Date.parse('1989-10-01'),
+        'gender_additional_information' => nil,
+        'gender_id' => gender.id,
+        'ethnicity_id' => ethnicity.id,
+        'last_synced_with_nomis' => be_within(2.seconds).of(now),
+      }
+    end
+    let(:expected_profile_attributes) do
+      {
+        'id' => nil,
+        'person_id' => nil,
+        'last_name' => 'JAVAS',
+        'first_names' => "EF'LIAICO MATTHEWS BARNARDOS",
+        'date_of_birth' => Date.parse('1989-10-01'),
+        'aliases' => nil,
+        'gender_id' => gender.id,
+        'ethnicity_id' => ethnicity.id,
+        'nationality_id' => nil,
+        'created_at' => nil,
+        'updated_at' => nil,
+        'assessment_answers' => [],
+        'gender_additional_information' => nil,
+        'latest_nomis_booking_id' => 1_093_139,
+        'last_synced_with_nomis' => be_within(2.seconds).of(now),
+      }
+    end
+
+    it 'initializes a new Person' do
+      person = service.call.person
```

Comment:
I think the service should return a person, not a profile.

Reply:
I thought the same. The caller saves the profile and due to relationships (i.e. a `Profile` instantiates a #person member variable) the Profile is the thing that needs saving against it.

Reply:
We'll be redoing all this later on, too. Separating out person/profile

Reply:
👍